include/rdpq.h

```diff
@@ -468,6 +468,55 @@ inline void rdpq_set_yuv_parms(uint16_t k0, uint16_t k1, uint16_t k2, uint16_t k
         _carg(x1fx, 0xFFF, 12) | _carg(y1fx, 0xFFF, 0)); \
 })
 
+/** @brief Only keep/draw even lines while drawing to a buffer */
+#define RDP_INTERLACE_DRAW_ODD 1
+/** @brief Only keep/draw odd lines while drawing to a buffer */
+#define RDP_INTERLACE_DRAW_EVEN 0
+
+/**
+ * @brief Enable whether the RDP should only draw even or odd lines (for interlaced displays) (RDP command: SET_SCISSOR)
+ * 
+ * This function is used to configure an interlacing function the RDP internally has
+ * to only draw half the lines into a framebuffer (Z-buffer is interlaced as well 
+ * automatically).
+ * 
+ * This interlacing mode assumes that the framebuffer itself has both fields in it
+ * and it can work on any framebuffer size (320x240, 512x256, 640x480 etc.)
+ * 
+ * This interlacing mode saves half the RDP's fillrate since it skips half the lines
+ * to draw. This is very helpful for performance in high-res mode since the system
+ * is primairly fillrate constrained.
+ * 
+ * 
+ * @param[in]   draw_field      True if the RDP should only draw odd lines, otherwise only even lines are drawn
+ * 
+ */
+__attribute__((always_inline)) 
+inline void rdpq_enable_interlaced(int draw_field) { 
+    int32_t o = (draw_field == 1)? 1 : 0; 
```

Comment:
I would simply go with `draw_field & 1` which is faster, also I believe it has a more "expected behavior" if you pass a number like 122.